engine-tests/src/test/java/org/terasology/engine/persistence/typeHandling/reflection/TestModuleEnvironmentSandbox.java

```diff
@@ -53,7 +53,7 @@ public void findSubtypeOfGlobalClass() {
 
     @Test
     public void findSubtypeOfEngineClass() {
-        assertNotEmpty(sandbox.findSubTypeOf("engine:BlockComponent", Component.class));
```

Comment:
Is it a problem that we can no longer resolve components by their simple identifiers when trying to resolve subtypes via the module sandbox?